k9mail/src/main/java/com/fsck/k9/K9.java

```diff
@@ -452,7 +452,7 @@ public static void save(StorageEditor editor) {
         editor.putBoolean("useVolumeKeysForListNavigation", useVolumeKeysForListNavigation);
         editor.putBoolean("autofitWidth", autofitWidth);
         editor.putBoolean("quietTimeEnabled", quietTimeEnabled);
-        editor.putBoolean("notificationDuringQuietTimeEnabled", notificationDuringQuietTimeEnabled);
+        editor.putBoolean("notificationDuringQuietTimeEnabled", notificationDuringQuietTimeDisabled);
```

Comment:
You can't just save `notificationDuringQuietTimeDisabled` as "notificationDuringQuietTimeEnabled". It'll be super confusing for everyone looking at the saved data because the value won't match the setting name (the logic will be inverted). Again, I'm suggesting to keep the original variable name.